cluster-autoscaler/main.go

```diff
@@ -156,6 +156,8 @@ var (
 		"Should CA ignore DaemonSet pods when calculating resource utilization for scaling down")
 	ignoreMirrorPodsUtilization = flag.Bool("ignore-mirror-pods-utilization", false,
 		"Should CA ignore Mirror pods when calculating resource utilization for scaling down")
+	ignorePodsThatShouldBeTerminated = flag.Bool("ignore-pods-that-should-be-terminated", false,
+		"Should CA ignore pods that should be terminated but are still present due to their attachment to a notReady node for scaling down")
```

Comment:
I did not use `Terminating` in the first place because, `Terminating` status is not enough: the pod must have run over its `TerminationGracePeriod`. In other words it should have gone through its terminating phase and should now be terminated/deleted.

What about:
```
Should CA ignore pods that have Terminating status and that have run over the TerminationGracePeriod but are still present due to their attachment to a notReady node for scaling down
```